jtrack/main.py

```diff
@@ -218,7 +222,7 @@ def interactive():
                         required=True)
     parser.add_argument('-s', '--summary', help='Value for the summary field.', dest='summary', required=True)
     parser.add_argument('-d', '--description', help='Value for the description field.', dest='desc')
-    parser.add_argument('-a', '--attachment', help='Path of file to add as attachment.', type=attachment_arg,
+    parser.add_argument('-a', '--attachments', help='Paths of files to add as attachment seperated by comma', type=attachment_arg,
```

Comment:
Can you keep it as "attachment" not to break old installations (although it should have been in plural)?

Reply:
I changed it by myself.